addons/mail/static/src/messaging/component/attachment_box/attachment_box.js

```diff
@@ -62,6 +62,24 @@ class AttachmentBox extends Component {
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onAttachmentCreated(ev) {
+        ev.stopPropagation();
+        this.trigger('o-attachments-changed');
```

Comment:
Why stop the propagation and trigger a new event? Can't just handle the original event in the parent instead?

Reply:
Not exactly because an attachment can be created from outside attachment box (like in composer for example) and in that case, the parent should not handle it